app/domain/authentication/authn_jwt/validate_restrictions_one_to_one.rb

```diff
@@ -0,0 +1,19 @@
+module Authentication
+  module AuthnJwt
+
+    # This class is responsible for retrieving the correct value from the JWT token
+    # of the requested attribute.
+    class ValidateRestrictionsOneToOne
```

Comment:
why is this not a command class?

Reply:
Because it is used in the generic resource restriction validatior done by @abrahamko 

Each authenticator gcp for example have file authenticator_request.rb that impliments validation of single restriction (annotation)

For example in gcp it here :
https://github.com/cyberark/conjur/blob/master/app/domain/authentication/authn_gcp/authentication_request.rb

The usage is here:
https://github.com/cyberark/conjur/blob/e44eaaef953651ebb2458cad6ef69bf6b1a1f040/app/domain/authentication/resource_restrictions/validate_resource_restrictions.rb#L71